packages/libs/sdk/src/api/types/nfts/getByWalletAddress.ts

```diff
@@ -15,9 +15,7 @@ export type WalletNFTsByAddressQueryType = {
 // Using the generated CodegenEthMainnetWalletNFTsByAddressQueryVariables as a base for the type here
 // since the variables will be the same for each query
 export type WalletNFTsByAddressQueryVariablesType =
-  CodegenEthMainnetWalletNFTsByAddressQueryVariables & {
-    contractAddresses?: string[];
-  };
+  CodegenEthMainnetWalletNFTsByAddressQueryVariables;
```

Comment:
This was a small change that I missed from https://github.com/quiknode-labs/qn-oss/commit/42a66271d29eeca6c6ee2fb1c8efc251bc0f2699, I noticed in the intellisense options when testing out the built package, it's not a valid option.